### PR TITLE
adding queuename

### PR DESCRIPTION
**Pull request by dgkanatsios**
This PR adds information about retrieving the matchmaking queue name (preview feature).

### CODE REVIEW

File: UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt [hunk 1]
```diff
@@ -15,11 +15,11 @@ MonoBehaviour:
   m_PixelRect:
     serializedVersion: 2
     x: 0
-    y: 42.22222
-    width: 1706.6666
-    height: 869.7778
+    y: 42
```
Comment (yegarci) [inline UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt:18]: Why have many values here changed? Where are they coming from?

File: MatchmakeSample/MatchmakeSample/Program.cs [hunk 2]
```diff
@@ -104,8 +104,28 @@ private static async Task CreateMatchmakeTicket(Player player, string mmQueueNam
...
                 QueueName = mmQueueName,
                 GiveUpAfterSeconds = 15,
-                Creator = new MatchmakingPlayer { Entity = new PlayFab.MultiplayerModels.EntityKey { Id = player.context.EntityId, Type = player.context.EntityType } } // Why isn't this just the caller?
+                Creator = new MatchmakingPlayer
+                {
+                    Entity = new PlayFab.MultiplayerModels.EntityKey { Id = player.context.EntityId, Type = player.context.EntityType }
+                }
             };
+
+            // uncomment and edit the following lines if you want to use Multiplayer Servers allocation
```
Comment (yegarci) [inline MatchmakeSample/MatchmakeSample/Program.cs:113]: This comment makes it seem like the below should be commented by default?

File: MatchmakeSample/matchMakingAndMps.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+You can have Matchmaking integrate and allocate servers on PlayFab Multiplayer Servers, you can see the documentation [here](https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/matchmaking/multiplayer-servers).
+
+## Preview feature - getting the matchmaking queue name after a server allocation
+
+We have recently launched a preview feature that allows you to get the matchmaking queue name after a server allocation. The way to retrieve the queue name is to search the config dictionary for the "PF_MATCH_QUEUE_NAME" string **after server is allocated**.
+
+### C#
+
+ - Make sure you grab the latest version of the C# GSDK
```
Comment (yegarci) [inline MatchmakeSample/matchMakingAndMps.md:11]: Can the link be added here?

**Review by yegarci — CHANGES_REQUESTED** [review]

File: UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt (hunk 1 above)
Comment (dgkanatsios) [reply UnityMirror/UnityServer/UserSettings/Layouts/default-2021.dwlt:18]: this is Unity version upgrade

File: MatchmakeSample/MatchmakeSample/Program.cs (hunk 2 above)
Comment (dgkanatsios) [reply MatchmakeSample/MatchmakeSample/Program.cs:113]: ouch, thx

File: MatchmakeSample/matchMakingAndMps.md (hunk 3 above)
Comment (dgkanatsios) [reply MatchmakeSample/matchMakingAndMps.md:11]: wanted to do it after we would push the 1.0.0 version, but the URL defaults to the latest one, so we can use that. Thanks!

**Review by yegarci — APPROVED** [review]
:shipit:

**Pull request merged**